backend/src/seeds/graphql/queries.ts

```diff
@@ -177,6 +177,40 @@ query ($currentPage: Int = 1, $pageSize: Int = 5, $order: Order = DESC, $statusF
   	contributionCount
     contributionList {
       id
+      firstName
+      lastName
+      amount
+      memo
+      createdAt
+      confirmedAt
+      confirmedBy
+      contributionDate
+      state
```

Comment:
Please rename this everywhere to `status`. We call it `status` in the backend, so why change the name in the frontends?

Reply:
We can change it like ``contributionDate`` to ``date`` and many more also.
But it was state and to make the less changes for the frontend I let it to state.

Reply:
I would do this in this Ticket: #2671